test/frozendict_only.py

```diff
@@ -41,3 +42,11 @@ def test_init(self, fd):
     def test_del_empty(self):
         fd = self.FrozendictClass({1: 2})
         assert fd.delete(1) is self.FrozendictClass()
+
+    def test_load_core_pickle(self):
```

Comment:
The same before: good for one shot tests, not for unit tests

Reply:
Similar to above, I think it is important to have a check that my `src/frozendict/core.py` compatibility code actually works.  I can improve this test if you want, as I do agree hard coding some pickle bytes is not ideal.

Reply:
> I can improve this test if you want, as I do agree hard coding some pickle bytes is not ideal.

Well, let's see :)